### PR TITLE
Activity card

### PR DESCRIPTION
**Pull request by Akshay-Jak**
### Functionality:
Added user email and user photo to activity card

### Solution:
added email address and photo URL to each

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

image

### CODE REVIEW

**Comment by viv9k** [comment]
The design of the card is not good please connect with me

**Review by viv9k — CHANGES_REQUESTED** [review]
It's not clear, please update only the changed lines. These files are having full line of changes..

**Review by viv9k — CHANGES_REQUESTED** [review]
unable to review it as the changes are show full file change, fix it and then raise it again.

**Comment by k0malSharma** [comment]
Please resolve the merge conflicts ASAP

File: src/app/body/task-details/task-details.component.ts [hunk 1]
```diff
@@ -75,13 +75,16 @@ export class TaskDetailsComponent implements OnInit {
   getActivityData() {
     var documentName = 'Organizations/' + this.orgDomain + '/Activity/' + this.Id + '/Action';
     this.tasksCollection = this.db.collection<Activity>(documentName);
+    console.log(2);
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:78]: Remove this logging statement

File: src/app/body/task-details/task-details.component.ts [hunk 2]
```diff
@@ -75,13 +75,16 @@ export class TaskDetailsComponent implements OnInit {
...
     this.activityData = this.tasksCollection.snapshotChanges().pipe(
       map(actions => actions.map(a => {
+        console.log(2);
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:81]: Remove this logging statement

File: src/app/body/task-details/task-details.component.ts [hunk 3]
```diff
@@ -75,13 +75,16 @@ export class TaskDetailsComponent implements OnInit {
...
         const data = a.payload.doc.data() as Activity;
         const id = a.payload.doc.id;
         return { id, ...data };
       }))
     );
+    console.log(this.activityData);
```
Comment (viv9k) [inline src/app/body/task-details/task-details.component.ts:87]: Remove this logging statement

**Review by viv9k — CHANGES_REQUESTED** [review]
Remove unused logging statements

**Review by viv9k — APPROVED** [review]
Ship it

**Review by viv9k — APPROVED** [review]
Ship it.

**Pull request merged**